javascript/javascript1/week3/CactusIO-interactive.js

```diff
@@ -0,0 +1,82 @@
+let activities =[];
+let limit = 180;
+let value = 0;
+var options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+var today  = new Date();
```

Comment:
`options` and `today` are unused and defined with `var`. Remember to clean-up before submitting! 😄